relays/bin-substrate/src/cli/bridge.rs

```diff
@@ -155,6 +70,20 @@ pub trait CliBridge: CliBridgeBase {
 	>;
 }
 
+pub trait MessagesCliBridge: CliBridgeBase {
+	const ESTIMATE_MESSAGE_FEE_METHOD: &'static str;
```

Comment:
Could you, please, add some docs for both trait and its associated consts/types + methods? Thanks! :)

Reply:
Sorry, I forgot. Done.

Reply:
That's fine :)